.github/profile/README.md

```diff
@@ -0,0 +1,112 @@
+# Welcome to Eclipse Open Regulatory Compliance Working  <img src="https://avatars.githubusercontent.com/u/189788385?s=200&v=4" width="200px"/>
```

Comment:
Any chance we can get a nicer banner, here, @shanda-eclipse?

Reply:
@tobie do you just need a background image for text to overlay? We have this but I can also make something else if you provide some further direction/specs.
![IMG_3105](https://github.com/user-attachments/assets/44a4b6b5-78f2-42cc-bc77-406623fcd2c2)

Reply:
This is a markdown document. You can't overlay stuff.

Reply:
@jrico-eclipse here is the banner we discussed 
![ORC Banner - general (2)](https://github.com/user-attachments/assets/e3e706de-2225-4242-84cd-a9dbe517c447)